statsmodels/discrete/tests/test_sandwich_cov.py

```diff
@@ -451,6 +451,70 @@ def test_basic(self):
         assert_allclose(res1.params, res2.params, rtol=rtol)
         assert_allclose(res1.bse, res2.bse, rtol=1e-10)
 
+    def test_score_hessian(self):
+        res1 = self.res1
+        res2 = self.res2
+
+        # We need to fix scale in GLM and OLS,
+        # discrete MLE have it always fixed
+        if isinstance(res2.model, OLS):
+            kwds = {'scale': res2.scale}
+        else:
+            kwds = {}
+        if isinstance(res2.model, OLS):
+            sgn = + 1
+        else:
+            sgn = -1  # see #4714
+
+        score1 = res1.model.score(res1.params * 0.98, scale=res1.scale)
+        score2 = res2.model.score(res1.params * 0.98, **kwds)
+        assert_allclose(score1, score2, rtol=1e-13)
+
+        hess1 = res1.model.hessian(res1.params, scale=res1.scale)
+        hess2 = res2.model.hessian(res1.params, **kwds)
+        assert_allclose(hess1, hess2, rtol=1e-10)
+
+        if isinstance(res2.model, OLS):
+            # skip the rest
```

Comment:
half a sentence on why OLS gets skipped?

Reply:
OLS does not have score_factor, hessian_factor
only GLM and now some discrete

Reply:
That will make a great addition to the comment here